src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -51,7 +66,13 @@ suspend fun blockingApplicationLogic(applicationState: ApplicationState) {
     }
 }
 
-fun Application.initRouting(applicationState: ApplicationState) {
+@KtorExperimentalAPI
```

Comment:
Hvor bleeding edge er dette? 😂

Reply:
Haha 😂 Det er vel egentlig bare ansvarsfraskrivelse. Behovet for den trigges på grunn av at vi bruker request.authorization() i klienten, og det ser virkelig ikke spesielt eksperimentelt ut når man ser hva den faktisk gjør :)